src/test/resources/testng.xml

```diff
@@ -815,12 +815,19 @@
       <class name="test.thread.parallelization.ParallelByMethodsTestCase8Scenario1"/>
     </classes>
   </test>
+
   <!--This test depends on System properties. So its intentionally being run at the end
   because the toggling of the system property will affect other tests -->
   <test name = "RunAtLast">
     <classes>
       <class name="test.dataprovider.issue128.DataProviderParametersMismatchTest"/>
     </classes>
   </test>
+
+  <test name="RunOrderingTest">
```

Comment:
Now, we may add a programmatically way to configure the test without system properties.